js/redux/modules/companyReducer.js

```diff
@@ -116,25 +123,49 @@ export const getCompanyProjects = (companyID, projectNumber) => dispatch => {
   database
     .doc("companys/" + companyID + "/projects/" + projectNumber)
     .get()
-    .then(function (doc) {
+    .then(function(doc) {
       let projectInfo = doc.data();
       dispatch(getProjectsInfo(projectInfo));
       dispatch(getDataError(null));
     })
-    .catch(function (error) {
+    .catch(function(error) {
       dispatch(getDataError(error));
     });
 };
+
+export const getProjectResources = (companyID, projectID) => dispatch => {
+  database
+    .collection(
+      "companys/" + companyID + "/projects/" + projectID + "/resources"
+    )
+    .get()
+    .then(collection => {
+      const resources = [];
+      collection.forEach(doc => {
+        resources.push(doc.data());
+      });
+      dispatch(getResources(resources));
+    })
```

Comment:
a better way to from line 142 to 148.

```
.then(collection => (
      collection.map(doc => doc.data())
    ))
.then((resources) => dispatch(getResources(resources));)
```

try this and let me know

Reply:
This does not work. I wonder if we need to define resources?